src/core/audio/AudioSoundIo.cpp

```diff
@@ -225,7 +218,7 @@ void AudioSoundIo::startProcessing()
 
 	if (! m_outstreamStarted)
 	{
-		if ((err = soundio_outstream_start(m_outstream)))
+		if (int err = soundio_outstream_start(m_outstream))
```

Comment:
This one doesn't have the `; err` like the others.

Reply:
It doesn't need to, so I'll remove the ones that do have `; err`.